### DISCUSSION TITLE
Dancers and kissers not blobby enough

### DISCUSSION THREAD

**Discussion by cmcaine** [C1710/blobmoji#66] [Style]
Some of these emoji are gendered non-blobs and that is awful.

All emoji should be genderless blobs or objects or whatever.

Blobby dancer: https://emojipedia.org/google/android-6.0.1/dancer/

Blobby kissers: https://emojipedia.org/google/android-5.0/kiss/

(Android 6.0.1 made the default kiss gendered, but I think that's actually a bit rubbish: https://emojipedia.org/google/android-6.0.1/kiss/)

Ok, so the boy and girl gendered blobs are actually real cute:

https://emojipedia.org/google/android-5.0/boy/
https://emojipedia.org/google/android-5.0/girl/

The person-frowning blob is also fantastic:

https://emojipedia.org/google/android-6.0.1/person-frowning/

Edit to add:

Anyway, I know there are some issues in that the blobs aren't available for the full range of emoji, but I think it would be neat if where they are available they were used.

We like the blobs because they are cute abstractions of people. The blob people emoji are cuter and often easier to read than the people emoji that are in the set now and are one of the things blob-enthusiasts probably want from this font

Thank you for your consideration of this critical issue.

**Comment by C1710** [comment]
I do agree that using the Blobs as the genderless version is a nice idea.  
However I also understand that some sort of consistency should be achieved, so in the end _all_ genderless versions should probably get a Blob style in the long run.  
BUT I'm Not Very Good™ at designing Blobs, so currently I can't promise anything...

**Comment by C1710** [comment]
About the frowning Blob: I don't think that everybody would understand that it's intended to be the same emoji as on other platforms, as it is _very_ abstract, so at least without some modifications it might be an issue to include it.

**Comment by Madis0** [comment]
For the frown I would suggest simply making it a tall oval shape like the other blobs OP mentioned. 

Shapes should be consistent like Android 7 did, even if they are more "blobby" when they represent persons (as opposed to generic expressions).

**Comment by Madis0** [comment]
List of emoji to blobify: https://emojipedia.org/neutral/ 

Notably superhero, supervillain and zombie are already blobs, though not the tall ones.

**Comment by lolbot-iichan** [comment]
I agree with OP, some emojis would benefit from reverting from 7.0 to 6.0.1 version.
I'd suggest converting all non-skin-colored emojis to 6.0.1-like blobs while keeping skin-colored versions similar to 7.0.

Just look how beautiful and blobby the police officer, the dancer, bowing person and person tipping hand were in 6.0.1 and how awfully humanlike are they since 7.0: https://blog.emojipedia.org/content/images/2016/08/android-7-human-emojis-emojipedia.png 

The same applies to families, kissing pairs, sport activities - they all were awesome and abstract in 6.0.1 and excessively human-specific and unblobby in 7.0: https://blog.emojipedia.org/content/images/2016/08/android-7-nougat-emojipedia-humans-2.jpg

Since the thread is marked as "opinions wanted", I express my opinion that this emoji set would gain a lot from bringing some 6.0.1 designs back.

P.S. You and contributors have done a great job on expanding 7.0 set to Unicode 14 standard, adding various types of jobs, activities and emotions. Thank you!

![изображение](https://user-images.githubusercontent.com/5786428/134788180-8ef886cf-0829-4808-ae4a-7bdbb87a89a3.png)

**Reply by Secret-chest** [reply]
I think families should be left human.

**Comment by Secret-chest** [comment]
> I'd suggest converting all non-skin-colored emojis to 6.0.1-like blobs while keeping skin-colored versions similar to 7.0.

A better way would be to convert all genderless versions to those.
